.flake8

```diff
@@ -1,3 +1,3 @@
 [flake8]
-ignore = E125,D100,D101,D102,D103,D105,D106,D107,D200,D202,D211,P101,FI15,FI16,FI12,FI11,FI17,FI50,FI53,FI54,MQ101,T000
+ignore = E125,D100,D101,D102,D103,D105,D106,D107,D200,D202,D211,D403,P101,FI15,FI16,FI12,FI11,FI17,FI50,FI53,FI54,MQ101,T000
```

Comment:
I'm not a huge fan of constantly changing these.. any reason this is here?

Reply:
I'm adding it here because I also add it to prospector.

`flake8` is used in `pre-commit` command and also in the IDE linter (emcas, vim, atom, pycharm, etc) if you have it configured.

We need a consistency between prospector and flake8.